server.R

```diff
@@ -747,6 +635,140 @@ server <- function(input, output, session) {
     }
   })
 
+  observeEvent(input$reset, {
+    updateSelectInput(session,
+      "sector",
+      choices = sectors_v,
+      selected = "Construction"
+    )
+
+    updateSelectInput(session,
+      "region",
+      choices = regions_v,
+      selected = "England"
+    )
+  })
+
+  observe({
+    if (input$subsectorlevel == "Subject and qualification") {
+      # Render the specific ui elements for the sidebar when the Subject and
+      # Qualification tab is selected.
+      output$SubjQualInputPanel <- renderUI({
+        tagList(
+          helpText("Choose an industry sub-sector and qualification level to view further detail on subject and qualification data."),
+          br(),
+          br(),
```

Comment:
@rmbielby could you please remove one of these br() as it shows too larger gap between help_text() and the selectiseInput()?

Reply:
Done and pushed.